haystack/nodes/file_converter/base.py

```diff
@@ -192,14 +198,20 @@ def run(  # type: ignore
 
     def run_batch(  # type: ignore
         self,
-        file_paths: Union[Path, List[Path]],  # type: ignore
-        meta: Optional[Union[Dict[str, str], List[Optional[Dict[str, str]]]]] = None,  # type: ignore
-        remove_numeric_tables: Optional[bool] = None,  # type: ignore
-        valid_languages: Optional[List[str]] = None,  # type: ignore
+        file_paths: Union[Path, List[Path]],
+        meta: Optional[Union[Dict[str, str], List[Optional[Dict[str, str]]]]] = None,
+        remove_numeric_tables: Optional[bool] = None,
+        known_ligatures: Dict[str, str] = KNOWN_LIGATURES,
```

Comment:
Do I get it right that these parameters were also missing ? (Additionally to the `id_hash_keys` ?)

Reply:
Okay, thanks!